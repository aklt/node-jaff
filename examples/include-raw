#begin coffee
name = 'raw'
#end
Include interpreted
#include include/{name}
Include raw
#include !include/{name}
