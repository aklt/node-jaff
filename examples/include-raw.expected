Include interpreted
Hier ist ein Schmetterling.
Include raw
#begin coffee
val = 'Schmetterling'
#end
Hier ist ein {val}.
